src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -327,6 +327,8 @@
    jruby-service :- (schema/protocol jruby-protocol/JRubyPuppetService)]
   (let [format-task (fn [task-object]
                       {:name (:name task-object)
+                       :private (get (:metadata task-object) :private false)
+                       :description (get (:metadata task-object) :description "")
```

Comment:
Would probably be cleaner to use `get-in` here, e.g. `(get-in task-object [:metadata :private] false)`

Reply:
Ah, yeah that makes more sense.